cypress/e2e/CMS/WYSIWYG/$EmbedFeaturedContent.feature

```diff
@@ -0,0 +1,201 @@
+Feature: As a cms user I want to be able to embed Featured contents in Article content type
+
+    Scenario: User is adding new test Article content type for embedding it as a Featured Content
+        Given user is navigating to "/user/login"
+        When user enters credentials
+        And user clicks "Log in" button
+        Then user is logged in and the user name "admin" is displayed in the toolbar
+        And the tool bar appears at the top
+        When user clicks on "Content" tab
+        And user clicks on "Add content" action button
+        And user clicks on "Article" content type
+        Then page title is "Create Article"
+        When user selects test site section
+        And user fills out the following fields
+            | fieldLabel               | value                                                                | field_name                     |
+            | Pretty URL               | test-article-for-testing                                             | field_pretty_url               |
+            | Page Title               | Automated Test Article for Featured Content Test                     | title                          |
+            | Browser Title            | Automated Test Article for Featured Content Test - Browser Title     | field_browser_title            |
+            | Card Title               | Automated Test Article for Featured Content Test - Card Title        | field_card_title               |
+            | Meta Description         | Automated Test Article for Featured Content Test - Meta Description  | field_page_description         |
+            | Feature Card Description | Automated Test Article for Featured Content Test - Feature Card Desc | field_feature_card_description |
```

Comment:
remove feature card description from here - blog posts will have description and article wont ( 2 different use cases)

Reply:
done